### ISSUE TITLE
New `Event` page needed

### ISSUE THREAD

**Issue by Jake3231** [UTDNebula/jupiter#107]
# Overview

As a user, I should be able to select an event and view detailed event information. This Issue is tracking work to bring up a new `Event` page that can show detailed information about an org's event.

Likely, this should include some sort of image, title, description, date/time, interested participants count, and possibly more.

# Proposal

- [ ] Create new Event page that is accessed by clicking an event in the Events browser
- [ ] Create draft layout for key information
- [ ] Allow the page to be populated with non-placeholder data when sample events are clicked.

**Comment by Master7130** [comment]
Can we rename the events directory page to be called "eventsDirectory" from "events." This would reduce confusion between the page I'm going to be creating, "event." Or we could call this page something else.

**Comment by Jake3231** [comment]
@Master7130 I may have missed a previous comment, were you saying that the naming of the page might be unclear?

**Comment by Master7130** [comment]
Yeah that was my initial thought and then I realized I could just make a folder containing the events page and the dynamic route for each event like for the directory.

**Comment by Master7130** [comment]
Someone seems to have created an events page already. What would be the next steps for this? Pull data from firebase to populate it instead of dummy data?

**Comment by RubenOlano** [comment]
I think that’d be a good idea. We can also extend events to include
descriptions and have a page for individual events

**Comment by Master7130** [comment]
Is this good to be closed?
